recipes/configs/llama2/7B_full.yaml

```diff
@@ -1,9 +1,27 @@
-# Config for FullFinetuneRecipe in full_finetune_distributed.py
+# Config for multi-device full finetuning in full_finetune_distributed.py
+# using a Llama2 7B model
 #
-# To launch, run the following command from root:
-#   tune --nnodes 1 --nproc_per_node 1 full_finetune_distributed \
-#   --config full_finetune_distributed \
-#   checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR> ...
+# This config assumes that you've run the following command before launching
```

Comment:
Why add this huge block in the config?

Reply:
Currently there's no documentation on the configs at all. Once we have live docs available, we can add these to the docs. But for now, I'd like to give some understanding to users about when and how to use each config.

Reply:
I think there's still some information in the README on configs and we can make that more clear. I think cluttering up the configs can be overwhelming.

Reply:
Why would it be overwhelming? Isn't it just documentation? I don't think we would be able to add config-level info to the README?